src/mixxxapplication.cpp

```diff
@@ -204,15 +204,19 @@ bool MixxxApplication::notify(QObject* pTarget, QEvent* pEvent) {
         QDebug debug = qDebug();
         debug << "Processing"
               << pEvent->type()
-              << "for object";
-        if (pEvent->type() == QEvent::DeferredDelete ||
-                pEvent->type() == QEvent::ChildRemoved) {
-            // pTarget can be already dangling in case of DeferredDelete
-            debug << static_cast<void*>(pTarget); // will print dangling address
+              << "for";
+        if (pTarget) {
+            if (pEvent->type() == QEvent::DeferredDelete ||
+                    pEvent->type() == QEvent::ChildRemoved) {
+                // pTarget can be already dangling in case of DeferredDelete
+                debug << static_cast<void*>(pTarget); // will print dangling address
+            } else {
+                debug << pTarget // will print address, class and object name
+                      << "running in thread:"
+                      << pTarget->thread()->objectName();
+            }
         } else {
-            debug << pTarget // will print address, class and object name
-                  << "running in thread:"
-                  << pTarget->thread()->objectName();
+            debug << "null pointer (THIS IS PROBABLY A BUG!)";
```

Comment:
This will still trigger a crash with debug assertions enabled, probably better than a segfault though since this isn't undefined behavior. We're already behind a developer flag, so the "verify" case will likely not come up very often

Reply:
> This will still trigger a crash with debug assertions enabled

Only if compiled with `DEBUG_ASSERTIONS_FATAL`. Regardless of whether this is behind the developer flag or not, if this is an indicator of a bug, we should make that explicit. This is exactly what the `DEBUG_ASSERT` facilities were made for. I don't understand how this case is different.

Reply:
I've updated it to use `VERIFY_OR_DEBUG_ASSERT`